src/os/ui/ogc/ogcserver.js

```diff
@@ -983,6 +1061,11 @@ os.ui.ogc.OGCServer.prototype.parseWfsCapabilities = function(response, uri) {
         descriptor.setKeywords(tags);
       }
 
+      if (descriptor instanceof os.data.BaseDescriptor) {
```

Comment:
Should we add the new functions to `IDataDescriptor` so we don't need this `instanceof` check? It's a breaking change, but seems cleaner than this.